lib/Parse/Lexer.cpp

```diff
@@ -353,8 +354,13 @@ void Lexer::skipToEndOfLine(bool EatNewline) {
     case 0:
       // If this is a random nul character in the middle of a buffer, skip it as
       // whitespace.
-      if (CurPtr-1 != BufferEnd) {
+      switch (getNulCharacterKind(CurPtr - 1)) {
+      case NulCharacterKind::Embedded:
         diagnoseEmbeddedNul(Diags, CurPtr-1);
+        LLVM_FALLTHROUGH;
+      case NulCharacterKind::CodeCompletion:
+        goto LoopStart;
```

Comment:
Can this be replaced with a `continue`?

Reply:
Oh, thanks.
I overlooked `continue` in `switch` is bound to outer `while` not to `switch` what is different from `break`.
Interesting...
I will fix it so.